docs/modules/deployment/pages/core/getting-started.adoc

```diff
@@ -139,8 +139,7 @@ The OpenNMS {page-component-title} core services run as an unprivileged user and
 For this reason, the default port for the SNMP Trap/Inform listener is set to port number 10162/udp instead of the IANA registered port number 162/udp.
 The following example shows how to configure the local firewall daemon to forward port 162/udp to 10162/udp.
 
-TIP: If you need SNMP Trap listener on port 162/udp directly, you can add the Linux `CAP_NET_BIND_SERVICE` capability to the Java binary using `setcap`.
-     Be aware: this method allows any Java program run on your system to bind to privileged ports < 1024.
+NOTE: If you need SNMP Trap listener on port 162/udp directly, see the "Binding to privileged ports" steps in the <<core/getting-started.adoc#set-up-core-instance, Setup the Core Instance>> section above.
```

Comment:
```suggestion
NOTE: If you need SNMP Trap listener on port 162/udp directly, see the "Binding to privileged ports" steps in <<core/getting-started.adoc#set-up-core-instance, Setup the Core Instance>>.
```
Title should actually be "Set up the Core Instance", since "set up" is a verb here, but it would need to change in xref and title.

Reply:
```suggestion
NOTE: If you need SNMP Trap listener on port 162/udp directly, see the "Binding to privileged ports" steps in <<core/getting-started.adoc#set-up-core-instance, Set up the Core Instance>>.
```